README.md

```diff
@@ -4,6 +4,7 @@ The Franklin based project for creative cloud pages on www.adobe.com. Based off
 ## Code merge process
 https://github.com/adobecom/cc/wiki
 
+
```

Comment:
What's the change here? I only see changes on minified dependency, is there any src change?

Reply:
It got there by mistake, thanks for noticing, I removed it